layers/gpu/resources/gpuav_shader_resources.h

```diff
@@ -79,11 +79,16 @@ struct BindingLayout {
     uint32_t count;
 };
 
-// For each descriptor index we have a 32-bit slot to mark what happend on the GPU.
-// Since most devices can only support 32 descriptor sets, we can start to be clever and compress info into these 32-bits by
-// exploiting the fact certain data we are saving doesn't need a full 32-bit to save. GLSL doesn't have bitfields and don't want to
-// make this structs in case we can't fit everything in 32-bits anymore.
-typedef uint32_t PostProcessDescriptorIndexSlot;
+struct PostProcessDescriptorIndexSlot {
+    // For each descriptor index we have a 32-bit slot to mark what happend on the GPU.
```

Comment:
```suggestion
    // For each descriptor index we have a 32-bit slot to mark what happend on the GPU.
```
This part is no longer true given your changes? Should be "... we have 64 bits ..."?

Reply:
the struct is 64-bit, but the first `32-bit` is still being sectioned off... will update comment a bit